.github/workflows/paportal-rolling-instance-actions.yml

```diff
@@ -109,6 +115,8 @@ jobs:
       WF_USERNAME: davidjen@ppdevtools.onmicrosoft.com
       WF_APPID: 8a7729e0-2b71-4919-a89a-c789d0a9720a
       WF_TENANT_ID: 3041a058-5110-495a-a575-b2a5571d9eac
+      PORTAL_WORKING_DIRECTORY: ./paportal-download
```

Comment:
nit: env.Portal_working_directory and ...manifest_file_path

Reply:
This job runs on linux, don't we have to reinitialize the variables? Or we can reuse env variables from the previous job?

Reply:
well, does your workflow work when you queue a flow for your PR branch?
Also: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#env

Reply:
Yes it is running as expected, here's a run against my branch, https://github.com/microsoft/powerplatform-actions/actions/runs/2230532791

Reply:
Will take up this in another PR if required